docs/reference/errors-and-warnings/NU3024.md

```diff
@@ -0,0 +1,47 @@
+---
+title: NuGet Warning NU3024
+description: NU3024 Warning code
+author: heng-liu
+ms.author: henli
+ms.date: 3/18/2019
+ms.topic: reference
+ms.reviewer: dtivel
+f1_keywords: 
+  - NU3024
+---
+
+# NuGet Warning NU3024
+
+## Scenario 1
+
+<pre>Package 'SamplePackage v1.0.0' from source 'http://source.com/index.json': The primary signature's timestamp certificate has an unsupported digest algorithm.</pre>
```

Comment:
Can you search and replace all instances of `source.com` with `contoso.com` both in your PR and in existing package signing docs?  Also check if `source.com` is used in other docs.

Reply:
Fixed it. Thanks.